includes/specials/SpecialGoToInterwiki.php

```diff
@@ -76,17 +76,19 @@ public function execute( $par ) {
 		}
 
 		$url = $target->getFullURL();
-		if ( !$target->isExternal() || ( $target->isLocal() && !$force ) ) {
-			// Either a normal page, or a local interwiki.
-			// Just redirect.
-			$this->getOutput()->redirect( $url, '301' );
-		} else {
+		/* Fandom change - start (PLATFORM-11316) */
+		$host = parse_url( $url, PHP_URL_HOST );
+		if ( $force || ( !str_contains( $host, '.fandom.com' ) && !str_contains( $host, '.wikia.com' ) && !str_contains( $host, '.gamepedia.com') ) ) {
```

Comment:
I would avoid such hardcoded changes especially in Mediawiki. Logic that checks if this is external link and non-Local should suffice.
How about override this via `InterwikiLookup`/`InterwikiLookupAdapter`? This should solve problem of "bad" data in those tables and as a followup we could enforce restrictions on UI side if needed.

Reply:
Actually there is a hook (albeit seems to be deprecated) `onInterwikiLoadPrefix` that we could use to override local flag for interwiki.